docs/experimental/extendable-param-files.md

```diff
@@ -0,0 +1,63 @@
+# Using the Extendable Bicep Parameters Feature (Experimental!)
+
+## What is it?
+
+Extendable Bicep Parameter Files is a feature that allows you to extend `.bicepparam` files from another `.bicepparam` file in order to reuse parameters across multiple deployments.
+
+When using extendable bicep parameter files, you will have a `base.bicepparam` file that can be used by multiple `.bicep` and `.bicepparam` files.
+
+The `base.bicepparam` file will link to a `shared.bicepparam` file, which will refer to a `main.bicep` file and your `base.bicepparam file`. In this case, you will be able to reuse and refer to this `base.bicepparam` file multiple times.
+
+## Using
+
+`main.bicep` This is your main bicep file, which will define your parameters for deployment.
+
+```bicep
+param foo string
```

Comment:
instead of `foo`s and `bar`s can we use more practical examples? One example we've seen a few times is having parameters like `location` or `namePrefix` in the base param file.